File: iOS/Example/Example/Models/Workout.swift
//
//  Workout.swift
//  Example
//
//  Created by Raffaele Cerullo on 23/05/2021.
//

import Foundation

struct Workout: Codable {
    let id: Int
    let name: String
    let duration: String
    let difficulty: String
    let description: String
    let imageUrl: String
}

File: Android/Example/app/src/main/java/com/example/fiitqatest/domain/Workout.kt
package com.example.fiitqatest.domain

data class Workout(
    val id: Int,
    val name: String,
    val duration: String,
    val imageUrl: String
)

data class WorkoutDetails(
    val id: Int,
    val name: String,
    val duration: String,
    val difficulty: String,
    val description: String,
    val imageUrl: String
)
